site/src/pages/components/enhanced-range-input.explainer.mdx

```diff
@@ -0,0 +1,475 @@
+}
+```
+**HTML:**
+
+```html
+<input type="range" min="0" max="100" value="50">
+```
+
+### Dual-Handle Range
+
+**CSS:**
+```css
+
+input[type="range"][handles="2"] {
+    appearance: base-range;
+    width: 300px;
+}
+
+input[type="range"][handles="2"]::range-track {
+    height: 6px;
+    background-color: #f0f0f0;
+}
+
+input[type="range"][handles="2"]::range-fill {
+    background-color: #4CAF50;
+}
+
+input[type="range"][handles="2"]::range-thumb {
+    width: 24px;
+    height: 24px;
+    background-color: #2196F3;
+    border-radius: 50%;
+}
+```
+
+**HTML**
+```html
+<input type="range" handles="2" min="0" max="1000" value="250,750" step="10">
+```
+
+### Range with Custom Tick Styling
+
+**CSS:**
+
+```css
+input[type="range"] {
+    appearance: base-range;
+    width: 300px;
+}
+
+input[type="range"]::range-track {
+    height: 4px;
+    background-color: #ddd;
+}
+
+input[type="range"]::range-fill {
+    background-color: #4CAF50;
+}
+
+input[type="range"]::range-thumb {
+    width: 20px;
+    height: 20px;
+    background-color: #2196F3;
+    border-radius: 50%;
+}
+
+/* Style all ticks */
+input[type="range"]::range-tick {
+    width: 2px;
+    height: 8px;
+    background-color: #999;
+}
+
+/* Style every fifth tick differently */
+input[type="range"]::range-tick:nth-child(5n) {
+    height: 12px;
+    background-color: #333;
+}
+
+/* Style all tick labels */
+input[type="range"]::range-tick-label {
+    font-size: 12px;
+    color: #666;
+}
+
+/* Style labels for every fifth tick differently */
+input[type="range"]::range-tick-label:nth-child(5n) {
+    font-weight: bold;
+    color: #333;
+}
+```
+
+**HTML:**
+
+```html
+<input type="range" min="0" max="100" step="5" list="percentages">
+<datalist id="percentages">
+    <option value="0" label="0%"></option>
+    <option value="25" label="25%"></option>
+    <option value="50" label="50%"></option>
+    <option value="75" label="75%"></option>
+    <option value="100" label="100%"></option>
+</datalist>
+```
+
+### Multi-Handle Range with Colored Segments
+
+**CSS:**
+
+```css
+input[type="range"][handles="3"] {
+    appearance: base-range;
+    width: 300px;
+    height: 20px;
+}
+
+input[type="range"][handles="3"]::range-track {
+    height: 10px;
+    background-color: #ddd;
+}
+
+input[type="range"][handles="3"]::range-thumb {
+    width: 20px;
+    height: 20px;
+    background-color: #2196F3;
+    border-radius: 50%;
+}
+
+input[type="range"][handles="3"]::range-segment:nth-child(1) {
+    background-color: #FF5733;
+}
+
+input[type="range"][handles="3"]::range-segment:nth-child(2) {
+    background-color: #33FF57;
+}
+
+input[type="range"][handles="3"]::range-segment:nth-child(3) {
+    background-color: #3357FF;
+}
+```
+
+**HTML:**
+
+```html
+<input type="range" handles="3" min="0" max="100" value="25,50,75">
+```
+## Considerations and Open Questions
+
+1. Should we support more than two handles, or limit to dual-handle ranges? If more, is there a maximum?
```

Comment:
A few of these have been discussed in issues, e.g. this one. Perhaps link to those, where they exist?

Reply:
I did add the previous research at the top of the page as Luke suggested. Not sure if that's enough? At the top of the page I did link a few discussions